csharp/ql/lib/semmle/code/asp/WebConfig.qll

```diff
@@ -7,67 +7,88 @@ import csharp
 /**
  * A `Web.config` file.
  */
-class WebConfigXML extends XMLFile {
-  WebConfigXML() { this.getName().matches("%Web.config") }
+class WebConfigXml extends XMLFile {
```

Comment:
Why is `XMLFile` not also changed?

Reply:
Ah, I guess that is because of what you mentioned in the `Leftovers` section.

Reply:
> Ah, I guess that is because of what you mentioned in the Leftovers section. 

Exactly. I'll probably look at that in a followup.